### PR TITLE
Fix docker container for postgresql

### LINKED ISSUE #1048: Running Huginn in a Docker container
Setting up Huginn via Docker.

Ran `docker pull cantino/huginn` then `docker pull postgres`.

Next, In the Docker instructions, this section shows how to link Huginn with the postgres container:

`docker run --rm --name huginn --link huginn_postgres:postgres -p 3000:3000 -e "HUGINN_DATABASE_USERNAME=huginn" -e "HUGINN_DATABASE_PASSWORD=pass@word" cantino/huginn`

Is huginn_postgres a constant variable or does it change based on individual setups?

I keep getting "Error response from daemon: Could not get container for huginn_postgres"

### PR DESCRIPTION
**Pull request by dsander**
This needs #1040 to work, fixes #1048

### CODE REVIEW

File: docker/scripts/init [hunk 1]
```diff
@@ -18,10 +18,10 @@ if [ -n "${MYSQL_PORT_3306_TCP_ADDR}" ]; then
   HUGINN_DATABASE_ADAPTER=${HUGINN_DATABASE_ADAPTER:-mysql2}
   HUGINN_DATABASE_HOST=${HUGINN_DATABASE_HOST:-${MYSQL_PORT_3306_TCP_ADDR}}
   HUGINN_DATABASE_PORT=${HUGINN_DATABASE_PORT:-${MYSQL_PORT_3306_TCP_PORT}}
-elif [ -n "${POSTGRESQL_PORT_5432_TCP_ADDR}" ]; then
-  HUGINN_DATABASE_ADAPTER=${HUGINN_DATABASE_ADAPTER:-postgres}
-  HUGINN_DATABASE_HOST=${HUGINN_DATABASE_HOST:-${POSTGRESQL_PORT_5432_TCP_ADDR}}
-  HUGINN_DATABASE_PORT=${HUGINN_DATABASE_PORT:-${POSTGRESQL_PORT_5432_TCP_PORT}}
+elif [ -n "${POSTGRES_PORT_5432_TCP_ADDR}" ]; then
+  HUGINN_DATABASE_ADAPTER=${HUGINN_DATABASE_ADAPTER:-postgresql}
```
Comment (cantino) [inline docker/scripts/init]: Was `postgres` a typo?

**Comment by cantino** [comment]
@ianblenke, mind reviewing?

File: docker/scripts/init (hunk 1 above)
Comment (dsander) [inline docker/scripts/init]: Yes.

**Comment by cantino** [comment]
@dsander, if this works for you, lets get it out.

**Comment by dsander** [comment]
@cantino This needs #1040 to be merged first to work.

**Comment by dsander** [comment]
Tested it again against the current master and linking to mysql/postgres as well as the build in mysql server works -> merging to fix the docker container :smile:

**Comment by cantino** [comment]
Nice!

**Pull request merged**